dom.bs

```diff
@@ -2585,6 +2612,9 @@ within a <var>parent</var>, run these steps:
  <li><p>If <var>node</var> is non-null, then <a for=/>insert</a> <var>node</var> into
  <var>parent</var> before null with the <i>suppress observers flag</i> set.
 
+ <li><p>If the <i>suppress observers flag</i> is unset, then <a>update slots</a> with
+ « <var>removedNodes</var> », <var>parent</var>, and « <var>addedNodes</var> ».
```

Comment:
This is the "replace all" algorithm that doesn't have a suppress observers flag at the moment so that would have to be added.